deployments/kubernetes/chart/reloader/templates/rbac.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+{{ include "reloader-labels.stakater" . | indent 4 }}
+{{ include "reloader-labels.chart" . | indent 4 }}
+  name: {{ template "reloader-name" . }}
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
```

Comment:
Do we need a cluster role? Isn't this supposed to run in the namespaces we want sir @rasheedamir ? I think we should restrict RBAC's to namespace level

Reply:
We've decided that the controller should only be able to run in 1 namespace that is specified in the env variable. Once that is done, we can easily implement all-namespace support keeping in mind the performance of the controller